cellxgene_schema_cli/cellxgene_schema/utils.py

```diff
@@ -50,6 +50,14 @@ def remove_deprecated_features(adata: ad.AnnData, deprecated: List[str]) -> ad.A
     return adata
 
 
+def get_num_vars_in_raw_matrix(adata: ad.AnnData) -> int:
```

Comment:
nit: I don't think its worth a util function for 1 line, same effect to just have this line + a comment in validate.py

Reply:
nvm--I see your note in the PR description to do this for easy mocking